Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml

```diff
@@ -291,7 +291,25 @@ script:
         - contextPath: Intezer.Analysis.SubAnalyses.Metadata
           description: A Sub Analysis metadata
           type: Unknown
-  dockerimage: demisto/py3-tools:0.0.1.30065
+    - name: intezer-get-analysis-iocs
+      arguments:
+        - default: false
+          description: The analysis ID we want to get the metadata for
+          isArray: false
+          name: analysis_id
+          required: true
+          secret: false
+      deprecated: false
+      description: Get analysis IOCs.
+      execution: false
+      outputs:
+        - contextPath: Intezer.Analysis.ID
+          description: The composed analysis ID
+          type: string
+        - contextPath: Intezer.Analysis.IOCs
+          description: The Analysis IOCs
+          type: Unknown
```

Comment:
Is the type really unknown?

Reply:
So would be better to write 'Dict' instead. 
(change it in the readme file as well).